codegen/projections/white_label/lib/white_label/config.rb

```diff
@@ -128,6 +128,13 @@ def validate_types!
       Hearth::Validator.validate_types!(validate_input, TrueClass, FalseClass, context: 'config[:validate_input]')
     end
 
+    def validate_range!
```

Comment:
I think I might argue for changing the name of the `validate_types!` method to be just `validate!` (requires changing Hearth::Configuration) and then adding the validations for each member together.  Eg:  
```
validate_type(config_1)
validate_range(config_1)
some_other_future_validation(config_1)

validate_type(config_2)
validate_tyupe(config_3)
```